### ISSUE TITLE
RkScript, set character variable

### ISSUE THREAD

**Issue by theludovyc** [rakugoteam/Rakugo-Dialogue-System#90]
[char_tag].[var_name] = value, variables or char_variables

**Comment by Jeremi360** [comment]
I'm not sure what this notation mean:
```gdscript
# user can do it both ways like in gdscript:
# 'dot' way
char_tag.var_name = value
# 'brackets' way
[char_tag][var_name] = value

# or this means I need to use both, if so, why?
[char_tag].[var_name] = value
```

**Comment by theludovyc** [comment]
Easy one, by exemple
sy.name = "Sylvie"

**Comment by Jeremi360** [comment]
> Easy one, by exemple sy.name = "Sylvie"

Okay, this is good way @theludovyc

**Comment by theludovyc** [comment]
> Okay, this is good way @theludovyc

Thank you sir o7

**Comment by theludovyc** [comment]
#95 help :)

**Comment by theludovyc** [comment]
#96 help :)

**Comment by Jeremi360** [comment]
It is a bit confusing those, 2 comments above:
> #95 help :) ; #96 help :) 

I read #95 and #96 already, so I know that you mean that those 2 will help this issue,
if I didn't read them, I would think that you need help with those issues, and somehow you are happy about need help.

**Comment by theludovyc** [comment]
> if I didn't read them, I would think that you need help with those issues, and somehow you are happy about need help.

I didn't think about that ! What word can I use so ?

**Comment by theludovyc** [comment]
Solved by #95
